OpenBSD patch for patch-src-Simplex-Messaging-Transport-HTTP2_hs.
Use defaultConfig from http2-5.4.0; the Config constructor is not accepted by
the OpenBSD build of the package API used here.
--- src/Simplex/Messaging/Transport/HTTP2.hs.orig
+++ src/Simplex/Messaging/Transport/HTTP2.hs
@@ -19,7 +19,7 @@
 import Data.Maybe (fromMaybe)
 import Foreign (mallocBytes)
 import Network.HPACK (BufferSize)
-import Network.HTTP2.Client (Config (..), defaultPositionReadMaker, freeSimpleConfig)
+import Network.HTTP2.Client (Config (..), defaultConfig, defaultPositionReadMaker, freeSimpleConfig)
 import qualified Network.HTTP2.Client as HC
 import qualified Network.HTTP2.Server as HS
 import Network.Socket (SockAddr (..))
@@ -40,7 +40,7 @@
   buf <- mallocBytes sz
   tm <- TI.initialize $ 30 * 1000000
   pure
-    Config
+    defaultConfig
       { confWriteBuffer = buf,
         confBufferSize = sz,
         confSendAll = cPut c,
